config/manager/manager.yaml

```
@@ -33,8 +33,8 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 30Mi
+            memory: 60Mi
```

Let's bump this some more. As it's the burst limit, it doesn't really make sense to keep it too low, as you pointed out.
```suggestion
            memory: 100Mi
```